mitosheet/src/utils/actions.tsx

```diff
@@ -1344,26 +1344,26 @@ export const createActions = (
             searchTerms: ['reset', 'index'],
             tooltip: "Resets a dataframe's index to 0,1,2,3... Removes current index entirely."
         },
-        // [ActionEnum.SNOWFLAKEIMPORT]: {
-        //     type: ActionEnum.SNOWFLAKEIMPORT,
-        //     shortTitle: 'Snowflake Import',
-        //     longTitle: 'Snowflake Import',
-        //     actionFunction: () => {
-        //         // We turn off editing mode, if it is on
-        //         setEditorState(undefined);
-
-        //         setUIState(prevUIState => {
-        //             return {
-        //                 ...prevUIState,
-        //                 currOpenTaskpane: {type: TaskpaneType.SNOWFLAKEIMPORT},
-        //                 selectedTabType: 'data'
-        //             }
-        //         })
-        //     },
-        //     isDisabled: () => {return undefined}, 
-        //     searchTerms: ['SQL', 'database', 'snowflake', 'import'],
-        //     tooltip: "Import dataframe from a Snowflake data warehouse"
-        // },
+        [ActionEnum.SNOWFLAKEIMPORT]: {
+            type: ActionEnum.SNOWFLAKEIMPORT,
+            shortTitle: 'Snowflake Import',
+            longTitle: 'Snowflake Import',
+            actionFunction: () => {
+                // We turn off editing mode, if it is on
+                setEditorState(undefined);
+
+                setUIState(prevUIState => {
+                    return {
+                        ...prevUIState,
+                        currOpenTaskpane: {type: TaskpaneType.SNOWFLAKEIMPORT},
+                        selectedTabType: 'data'
```

Comment:
The Snowflake Frontent should only be available if the user is on Enterprise ..... 

Should we update that?